sormas-ui/src/main/java/de/symeda/sormas/ui/utils/ShowDetailsListener.java

```diff
@@ -45,6 +49,7 @@ public ShowDetailsListener(String detailsColumnId, boolean showOnDoubleClick, Co
 		this.detailsColumnId = detailsColumnId;
 		this.showOnDoubleClick = showOnDoubleClick;
 		this.itemHandler = itemHandler;
+		this.lastHandlingTimestamp = DateTime.now().minusSeconds(1);
```

Comment:
just take "now", why subtracting a second?

Reply:
I got it: It's the initialization to avoid `null` check later. Maybe a code comment would help

Reply:
It's necessary so the columns can be clicked from the beginning on, and not just after 1 second has passed